app/main/routes.py

```diff
@@ -153,13 +149,13 @@ def quick_access():
     return render_template("quick-access.html")
 
 
-@bp.route("/record", methods=["GET"])
-def record():
-    return render_template(
-        "record.html",
-        consignment=consignment_response,
-        consignment_files=consignment_files_response,
-    )
+# @bp.route("/record", methods=["GET"])
```

Comment:
cant import whats used in this view but I know @colinbowen is working on this view so Im sure will be fixed in an upcoming PR